modules/virt-virtctl-commands.adoc

```diff
@@ -73,6 +73,18 @@ You can use `virtctl create` commands to create manifests for virtual machines,
 |`virtctl create vm --name <vm_name>`
 |Create a VM manifest, specifying a name for the VM.
 
+|`virtctl create vm --user <user_name> --ssh-key\|password-file=<value>`
+|Create a VM manifest with a cloud-init configuration to create the selected user and either add an SSH public key from the supplied string, or a password from a file.
+
+|`virtctl create vm --access-cred type:password,src:<secret>`
+|Create a VM manifest with a user and password combination injected from the selected secret.
+
+|`virtctl create vm --access-cred type:ssh,src:<secret>,user:<user_name>`
+|Create a VM manifest with an SSH public key injected from the selected secret.
+
+|`virtctl create vm --volume-sysprep src:<config_map>`
+|Create a VM manifest, specifying a ConfigMap to use as the sysprep volume. The ConfigMap must contain a valid answer file named `unattend.xml` or `autounattend.xml`.
```

Comment:
```suggestion
|Create a VM manifest, specifying a config map to use as the sysprep volume. The config map must contain a valid answer file named `unattend.xml` or `autounattend.xml`.
```

https://redhat-documentation.github.io/supplementary-style-guide/#config-map

Reply:
Thank you, that's a good catch. I fixed it with the last push.